### PR TITLE
GrateBalance

### PR DESCRIPTION
**Pull request by AveryThePanda**
Makes Grates cost only one rod, and returns the same amount. Catwalks still cost three, and return three.

### CODE REVIEW

**Comment by AveryThePanda** [comment]
!review

**Comment by SleepyWolfe** [comment]
a pretty grate balance!

File: html/changelogs/averythepanda-changelog.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Makes Grates cost only one metal rod, and it gives back one metal rod."
```
Comment (Geevies) [inline html/changelogs/averythepanda-changelog.yml:41]:
```suggestion
  - tweak: "Makes Grates cost only one metal rod, and it gives back one metal rod when deconstructed."
```

**Comment by Doxxmedearly** [comment]
@AveryThePanda Please address Geeves' suggestion and it'll be approved.

**Pull request merged**